.github/workflows/lint-workflow.yml

```diff
@@ -16,11 +18,11 @@ jobs:
 
     steps:
       - name: Clone
-        uses: actions/checkout@v3
+        uses: actions/checkout@v2
 
       - name: Lint
-        uses: DoozyX/clang-format-lint-action@v0.15
+        uses: DoozyX/clang-format-lint-action@v0.13
         with:
           source: "./"
           extensions: "h,c"
-          clangFormatVersion: 12.0.1
```

Comment:
The [app-builder](https://github.com/LedgerHQ/ledger-app-builder/) comes with [clang 12.0.1](https://pkgs.alpinelinux.org/package/v3.15/main/x86_64/clang-extra-tools), so it would be easier to keep it the same.